pylib/Stages/Provisioning/WWulf3.py

```diff
@@ -67,6 +67,14 @@ def execute(self, log, keyvals, testDef):
         # parse what we were given against our defined options
         cmds = {}
         testDef.parseOptions(log, self.options, keyvals, cmds)
+
+        mylog = {}
+        if cmds['target']: mylog['target'] = cmds['target']
+        if cmds['image']: mylog['image'] = cmds['image']
+        if cmds['controller']: mylog['controller'] = cmds['controller']
+        if cmds['bootstrap']: mylog['bootstrap'] = cmds['bootstrap']
```

Comment:
Perhaps for consistency you should split the above if statements into two lines like we do everywhere else?